pocs/state/states/default/parked.py

```diff
@@ -11,19 +11,35 @@ def on_enter(event_data):
             if pocs.should_retry is False or pocs.run_once is True:
                 pocs.say("Done retrying for this run, going to clean up and shut down!")
                 pocs.next_state = 'housekeeping'
-            else:
+            else:  # This branch will only happen if there is an error causing a shutdown
                 pocs.say("Things look okay for now. I'm going to try again.")
                 pocs.next_state = 'ready'
-        else:
+        else:  # Normal end of night
             pocs.say("Cleaning up for the night!")
             pocs.next_state = 'housekeeping'
     else:
+        pocs.say("No observations found.")
+        # TODO Should check if we are close to morning and if so do some morning
```

Comment:
Oh no, I've infected you with adding TODOs! ;-)

I'd love a similar behavior at the start of the evening (i.e. evening flats, or darks while the dome is closed).